src/components/Input/index.tsx

```diff
@@ -103,7 +103,7 @@ const Input = React.forwardRef<HTMLInputElement, InputProps>(
   ) => {
     const leftSideComponent =
       leftSideChild ?? convertTypeToComponent.left[`${type}`]
-    const rightSideComponent = rightSideChild ?? <CloseIcon />
+    const rightSideComponent = rightSideChild ?? <CloseIcon strokeWidth={0.6} />
```

Comment:
What was the reason for this?

Reply:
The stroke width is different for the two close icons so had to add that in to specify because I removed this stroke width from the icon itself